src/cljs/main/broadfcui/common.cljs

```diff
@@ -310,7 +309,9 @@
                   (if (:close-on-click props)
                     (.on element$ "click.zf.dropdown"
                          (fn [_]
-                           (.foundation element$ "close")))
+                           (js/setTimeout                   ; allow click handlers to fire
```

Comment:
Would slightly prefer:
```cljs
;; Allow click handlers to fire.
(js/setTimeout #(.foundation element$ "close") 0)
```
rather than comments on the right-hand margin, but happy either way.